apps/storefront/vite.config.ts

```diff
@@ -10,13 +10,20 @@ interface AssetsAbsolutePathProps {
   [key: string]: string;
 }
 
+const PRODUCTION_ENV = 'production';
+const TIER1_ENV = 'tier1';
+
 const assetsAbsolutePath: AssetsAbsolutePathProps = {
   staging: 'https://cdn.bundleb2b.net/b2b/staging/storefront/assets/',
   production: 'https://cdn.bundleb2b.net/b2b/production/storefront/assets/',
 };
 
 export default defineConfig(({ mode }) => {
-  const env = loadEnv(mode, process.cwd());
+  let modeVariable = mode;
+  if (mode === TIER1_ENV) {
+    modeVariable = PRODUCTION_ENV;
+  }
```

Comment:
const modeVariable = (mode === TIER1_ENV) ? PRODUCTION_ENV : mode;
Is it more concise and readable?